src/public/services/FormService.ts

```diff
@@ -0,0 +1,134 @@
+import axios from 'axios'
+
+import { FormUpdateParams } from 'src/app/modules/form/admin-form/admin-form.types'
+import { PublicFormViewDto } from 'src/app/modules/form/public-form/public-form.types'
+import {
+  FormMetaView,
+  IFormSchema,
+  IPopulatedForm,
+  PublicForm,
+} from 'src/types'
+import { FormFieldDto } from 'src/types/api'
+
+// endpoints exported for testing
+export const ADMIN_FORM_ENDPOINT = '/api/v3/admin/forms'
+export const PUBLIC_FORM_ENDPOINT = '/api/v3/forms'
+
+export const queryForm = async (): Promise<FormMetaView[]> => {
```

Comment:
I think I'll just go with `getDashboardView` too. I think this name is okay since the type `FormMetaView` is only specified for forms displayed on the dashboard.